podcasts/PodcastImageView.swift

```diff
@@ -30,6 +30,11 @@ class PodcastImageView: UIView {
     func setEpisode(_ episode: Episode, size: PodcastThumbnailSize) {
         guard let imageView = imageView else { return }
 
+        guard episode != currentEpisode else {
```

Comment:
I kind of wonder if this should go above the `imageView` unwrap but I don't think it really matters much.